pkg/epforwarder/epforwarder.go

```diff
@@ -11,6 +11,7 @@ import (
 	"sync"
 
 	"github.com/DataDog/datadog-agent/comp/logs/agent/config"
+	"github.com/DataDog/datadog-agent/pkg/aggregator/sender"
```

Comment:
Would it make sense to rename this import instead of the `logs/sender`?

Reply:
code updated.